portfolio/src/main/webapp/js/script.js

```diff
@@ -168,4 +172,40 @@ function addListItem(text, nodeParent) {
     nodeParent.appendChild(item);
 }
 
+function onKeyDown(event) {
```

Comment:
Will this submit the form if the user hits enter anywhere on screen? Is it possible to submit only if the text box is highlighted?

Reply:
onKeyDown() is registered with the onKeyDown event in the textarea element. I think this way it only handles the event when the textarea is selected (I can very well be wrong given my proficiency in HTML)

Reply:
Exactly! The eventListener is only added to the textarea, which means it would only submit the form once we press enter inside the text box. Thanks for the feedback

Reply:
sounds good